dependency-check-suppressions.xml

```diff
@@ -25,4 +42,525 @@
         <cve>CVE-2023-6481</cve>
     </suppress>
 
+    <suppress>
```

Comment:
I'm not sure it's worth doing each file given you have the wildcard suppressions above for spring and netty